Makefile

```diff
@@ -0,0 +1,63 @@
+NAME=cimgui-go Code Generator
+GOCMD=LC_ALL=C go
+TIMEOUT=5
```

Comment:
Why do you need TIMEOUT?

Reply:
I don't know to be honest.
It was in the Makefile I was basing on - will remove.